crates/constants/src/lib.rs

```diff
@@ -1,25 +1,32 @@
 use lazy_static::lazy_static;
 use serde::{Deserialize, Serialize};
-use std::collections::HashMap;
-
-#[derive(Debug, Serialize, Deserialize)]
-pub struct HardwareId {
-    pub body_id: String,
-    pub head_id: String,
-}
 
 pub const HULA_DBUS_INTERFACE: &str = "org.hulks.hula";
 pub const HULA_DBUS_PATH: &str = "/org/hulks/HuLA";
 pub const HULA_DBUS_SERVICE: &str = "org.hulks.hula";
 pub const HULA_SOCKET_PATH: &str = "/tmp/hula";
 pub const OS_IS_NOT_LINUX: bool = !cfg!(target_os = "linux");
 pub const OS_RELEASE_PATH: &str = "/etc/os-release";
-pub const OS_VERSION: &str = "7.5.6";
+pub const OS_VERSION: &str = "7.5.7";
 pub const SDK_VERSION: &str = "7.5.0";
 
+#[derive(Serialize, Deserialize)]
+pub struct Team {
+    pub team_number: u8,
+    pub hostname_prefix: String,
+    pub naos: Vec<Nao>,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct Nao {
+    pub number: u8,
+    pub body_id: String,
+    pub head_id: String,
+}
```

Comment:
These structs do not seem to be compatible with the team.toml you created and panics when accessed.

Reply:
oops... should be fixed now